test/createAzureResource.test.ts

```diff
@@ -93,6 +94,26 @@ suite('Create Azure Resources', async function (this: ISuiteCallbackContext): Pr
         assert.equal(createdApp.state, 'Running', `Web App state should be 'Running' rather than ${createdApp.state}.`);
     });
 
+    test('Configure Deployment Source to LocalGit', async () => {
+        let createdApp: WebSiteManagementModels.SiteConfigResource;
+        createdApp = await webSiteClient.webApps.getConfiguration(resourceName, resourceName);
+        assert.notEqual(createdApp.scmType, 'LocalGit', `Web App scmType's property value shouldn't be ${createdApp.scmType} before "Configure Deployment Source to LocalGit".`);
+        ext.ui = new TestUserInput([resourceName, 'LocalGit']);
+        await vscode.commands.executeCommand('appService.ConfigureDeploymentSource');
+        createdApp = await webSiteClient.webApps.getConfiguration(resourceName, resourceName);
+        assert.equal(createdApp.scmType, 'LocalGit', `Web App scmType's property value should be 'LocalGit' rather than ${createdApp.scmType}.`);
```

Comment:
Use these constants https://github.com/microsoft/vscode-azureappservice/blob/master/src/constants.ts#L25-L29 instead of typing in the string value 'None'.